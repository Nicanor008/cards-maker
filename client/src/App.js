import React from "react";
import { Provider } from 'react-redux';
import { BrowserRouter } from "react-router-dom";
import { Route, Switch } from "react-router";

import "react-bulma-components/dist/react-bulma-components.min.css";
import NavBar from "./components/nav";
import { store } from './store';
import HomePage from "./components/HomePage";
import Footer from "./components/nav/footer";
import CreateCards from "./containers/Cards/createCards";

function App() {
  return (
    <Provider store={store}>
      <BrowserRouter>
        <NavBar />
        <Switch>
          <Route exact path="/" component={HomePage} />
          <Route exact path="/create" component={CreateCards} />
        </Switch>
        <Footer />
      </BrowserRouter>
    </Provider>
  );
}

export default App;
