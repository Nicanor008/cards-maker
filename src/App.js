import React from 'react';
import './App.css';

import 'react-bulma-components/dist/react-bulma-components.min.css';
import { Button } from 'react-bulma-components';
import NavBar from './components/nav';
import SEO from './components/SEO';

function App() {
  return (
    <div>
      <div>
      <SEO title="Cards Maker" />
        <NavBar />
          <Button color="primary">My Bulma button</Button>
      </div>
    </div>
  );
}

export default App;
