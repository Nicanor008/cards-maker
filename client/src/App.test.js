import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import App from './App';

configure({ adapter: new Adapter() });

describe('App Component', () => {
it('should render page correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
})
