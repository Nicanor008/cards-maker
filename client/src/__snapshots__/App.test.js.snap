// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should render page correctly 1`] = `ShallowWrapper {}`;
